Project.toml

```diff
@@ -21,8 +21,8 @@ Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 
 [compat]
-EnsembleKalmanProcesses = "^0.5"
-julia = "~1.6"
+EnsembleKalmanProcesses = "^0.8"
+julia = "^1.6"
```

Comment:
cc @odunbar revert

Reply:
Reverted Julia but not EKP - we wish to have 0.8 from now on anyway